plugins/signals-backend/README.md

```diff
@@ -23,6 +23,7 @@ export default async function createPlugin(
     eventBroker: env.eventBroker,
     identity: env.identity,
     discovery: env.discovery,
+    permissions: env.permissions,
```

Comment:
worth rewriting the readmes for the new backend system - just a note for the future